### DISCUSSION TITLE
配置文件中的外部资源该怎么使用？

### DISCUSSION THREAD

**Discussion by iuyaa** [MetaCubeX/ClashMetaForAndroid#295] [General]
该导入什么样的文件，内容大概是什么
![Screenshot_2024-08-17-07-07-00-425_com.github.metacubex.clash.meta.jpg](https://github.com/user-attachments/assets/a034e1c1-fdf7-491e-b3f8-0a57653e3416)

**Comment by JayYu998** [comment]
请问解决了嘛
